src/app/_components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,48 @@
+'use client'
+
+import { useState } from 'react'
+
+import { Link } from '@phosphor-icons/react/dist/ssr'
+import * as Sentry from '@sentry/nextjs'
+import { useCopyToClipboard } from 'usehooks-ts'
+
+import { NOTIFICATION_DIALOG_DURATION_MS } from '@/constants/notificationDialogDuration'
+
+import { Icon } from '@/components/Icon'
+import { NotificationDialog } from '@/components/NotificationDialog'
+
+type CopyToClipboardProps = {
+  text: string
+}
+
+export function CopyToClipboard({ text }: CopyToClipboardProps) {
+  const [_, copy] = useCopyToClipboard()
```

Comment:
The `_` part can be removed if not used :)

```suggestion
  const [, copy] = useCopyToClipboard()
```